JebraAzureFunctions/JebraAzureFunctions/UserSignIn.cs

```diff
@@ -48,8 +48,8 @@ public static class UserSignIn
              */
 
             string courseIdS = Tools.ExecuteQueryAsync($"SELECT id FROM course WHERE code='{courseCode}'").GetAwaiter().GetResult();
```

Comment:
```suggestion
            string courseIdS = Tools.ExecuteQueryAsync($"SELECT id FROM course WHERE code={courseCode}").GetAwaiter().GetResult();
```

Since the `code` column is now an INT in the database, we shouldn't be wrapping it in single quotes here